### ISSUE TITLE
[pfto] torch.nn.functional.pad results in an wrong ONNX 

### ISSUE THREAD

**Issue by shinh** [pfnet/pytorch-pfn-extras#577]
Repro:

```
diff --git a/tests/pytorch_pfn_extras_tests/onnx_tests/test_export.py b/tests/pytorch_pfn_extras_tests/onnx_tests/test_export.py
index e7712ca..33cdcb5 100644
--- a/tests/pytorch_pfn_extras_tests/onnx_tests/test_export.py
+++ b/tests/pytorch_pfn_extras_tests/onnx_tests/test_export.py
@@ -167,3 +167,14 @@ def test_norm():
             return torch.norm(x)
 
     run_model_test(Net(), (torch.rand(2, 3, 5, 7),), opset_version=13)
+
+
+def test_pad():
+    class Net(torch.nn.Module):
+        def __init__(self):
+            super(Net, self).__init__()
+
+        def forward(self, x):
+            return torch.nn.functional.pad(x, pad=[2, 1, 3, 5], mode="constant", value=1.0)
+
+    run_model_test(Net(), (torch.rand(2, 3, 5, 7),))
```

Apparently, torch.onnx passes `padding` as a constant tensor to `constant_pad_nd` while pfto passes it as a scalar list.
